### PR TITLE
chore: specs-staging CODEOWNERS

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/celestia-app/issues/807

### CODE REVIEW

**Review by evan-forbes — APPROVED** [review]

**Comment by rootulp** [comment]
Will wait for approval from @adlerjohn b/c I don't want to merge things to `specs-staging` without his approval

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**
